repos/fdbt-site/src/pages/api/managePassengerGroup.ts

```diff
@@ -12,32 +12,30 @@ import { checkIntegerIsValid, removeExcessWhiteSpace } from './apiUtils/validato
 export const formatRequestBody = (req: NextApiRequestWithSession): GroupPassengerType => {
     const maxGroupSize = removeExcessWhiteSpace(req.body.maxGroupSize);
     const groupName = removeExcessWhiteSpace(req.body.passengerGroupName);
-    const passengerTypes: string[] = !req.body.passengerTypes
+    const passengerTypeNames: string[] = !req.body.passengerTypes
         ? []
         : isArray(req.body.passengerTypes)
         ? req.body.passengerTypes
         : [req.body.passengerTypes];
 
-    const companions: CompanionInfo[] = [];
-    if (passengerTypes.length > 0) {
-        passengerTypes.forEach((passengerType) => {
-            const minNumber = req.body[`minimumPassengers${passengerType}`];
-            const maxNumber = req.body[`maximumPassengers${passengerType}`];
-            const typeOfPassenger = req.body[`${passengerType}-type`];
-            const ageRangeMin = req.body[`${passengerType}-age-range-min`];
-            const ageRangeMax = req.body[`${passengerType}-age-range-max`];
-            const proofDocuments = req.body[`${passengerType}-proof-docs`];
-            companions.push({
-                name: passengerType,
-                passengerType: typeOfPassenger,
-                minNumber,
-                maxNumber,
-                ageRangeMin,
-                ageRangeMax,
-                proofDocuments,
-            });
-        });
-    }
+    const companions: CompanionInfo[] = passengerTypeNames.map((passengerTypeName) => {
+        const minNumber = req.body[`minimumPassengers${passengerTypeName}`];
+        const maxNumber = req.body[`maximumPassengers${passengerTypeName}`];
+        const typeOfPassenger = req.body[`${passengerTypeName}-type`];
```

Comment:
Could this be called 'passengerType' just to avoid needing to rename it on line 31?